tests/etl/test_doc_proc.py

```diff
@@ -0,0 +1,56 @@
+"""
+Test suite for document processing utilities.
+"""
+
+import os
+import unittest
+import tempfile
+import shutil
+from langchain_core.vectorstores import InMemoryVectorStore
```

Comment:
Nitpicking, but you can just accept the suggestion:

```suggestion
import os
import shutil
import tempfile
import unittest

from langchain_core.vectorstores import InMemoryVectorStore
```